tests/containers/bci_test.pm

```diff
@@ -65,7 +65,7 @@ sub reset_engines {
     my ($self, $current_engine) = @_;
     my ($version, $sp, $host_distri) = get_os_release;
     my $sp_version = "$version.$sp";
-    if ($sp_version =~ /15.3|15.4/) {
+    if ($sp_version =~ /15.3|15.4|15.5/) {
         # This workaround is only needed in SLE 15-SP3 and 15-SP4 (and Leap 15.3 and 15.4)
```

Comment:
and spř

Reply:
I'm sorry, I don't understand

Reply:
I guess he means that the comment is missing `SP5` :D

Reply:
I thought he means to add `15.6` and the ř is just a typo.

Reply:
I meant SP5 but now I see that it's there. Sorry.